### ISSUE TITLE
Failed to start container with v1.0.4

### ISSUE THREAD

**Issue by linux-china** [cdr/sail#215]
After upgrade to v1.0.4, and I got the following errors: 
```
2019-06-11 13:42:47 INFO writing sail proxy logs to /var/folders/fc/2bmlvpc166qbp2jc5d610jw00000gn/T/sailproxy_linux-china_sail-demo167613337
2019-06-11 13:42:47 ERROR        build run failed: failed to run container: failed to create container: Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /var/folders/fc/2bmlvpc166qbp2jc5d610jw00000gn/T/sail-code-server-cache/code-server
2019-06-11 13:42:47 ERROR        failed to remove linux-china_sail-demo
```
it's good with v1.0.3.

**Comment by nathanpotter** [comment]
Looking into this now, could you give some more details about the environment you're running in, and any other details you think may be helpful in debugging?

**Comment by linux-china** [comment]
My development env:
macOS Mojave 10.14.6
Docker desktop for Mac 2.0.4.1(34207)
Docker Cli version 19.03.0-beta3, build c55e026

and My demo is here https://github.com/linux-china/sail-demo

**Comment by nathanpotter** [comment]
Ok thank you, I was able to reproduce this as well on a macbook, I'll investigate and get a fix in to resolve this

**Comment by nathanpotter** [comment]
closed via #216 

Thanks for submitting the issue, I updated the release with a patch to fix this issue
